### DISCUSSION TITLE
✅ 🔗 Add Categories to Posts widget meta data (Duplicate)

### DISCUSSION THREAD

**Discussion by WPMonkey** [elementor/elementor#18315] [Feature Request]
Currently the Posts element does not have Categories in the meta data to display within the grid. That seems like a basic, critical, and often used piece of information that people want displayed within their post lists.

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the introduction of the Loop Builder in Elementor Pro version 3.9! 🥳 

You can now build your own Post cards with the Loop Grid widget, and use Dynamic Tags to show Post Terms.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
